.github/workflows/nightly_snapshot.yaml

```diff
@@ -286,67 +286,88 @@ jobs:
-            cp -a ../api/wasm-interpreter/pkg/* ./snapshots/$target_branch/wasm-interpreter/
-            git add snapshots/$target_branch/wasm-interpreter
-            git add -u snapshots/$target_branch/wasm-interpreter
+            rm -rf $output_path/wasm-interpreter
+            mkdir -p $output_path/wasm-interpreter
+            cp -a ../api/wasm-interpreter/pkg/* ./$output_path/wasm-interpreter/
 
-            rm -rf snapshots/$target_branch/editor
-            mkdir -p snapshots/$target_branch/editor
-            cp -a ../slintpad/* snapshots/$target_branch/editor/
-            git add snapshots/$target_branch/editor
-            git add -u snapshots/$target_branch/editor
+            rm -rf $output_path/editor
+            mkdir -p $output_path/editor
+            cp -a ../slintpad/* $output_path/editor/
 
-            rm -rf snapshots/$target_branch/docs
-            mkdir -p snapshots/$target_branch/docs
-            cp -a ../docs/site/* snapshots/$target_branch/docs
-            mkdir -p snapshots/$target_branch/docs/cpp
-            cp -a ../target/cppdocs/html/* snapshots/$target_branch/docs/cpp/
-            mkdir -p snapshots/$target_branch/docs/rust
-            cp -a ../target/doc/* snapshots/$target_branch/docs/rust/
+            rm -rf $output_path/docs
+            mkdir -p $output_path/docs
+            cp -a ../docs/site/* $output_path/docs
+            mkdir -p $output_path/docs/cpp
+            cp -a ../target/cppdocs/html/* $output_path/docs/cpp/
+            mkdir -p $output_path/docs/rust
+            cp -a ../target/doc/* $output_path/docs/rust/
 
             # Fix up link to Slint language documentation
-            sed -i "s!https://slint.dev/releases/.*/docs/!../../!" snapshots/$target_branch/docs/rust/slint/*.html
+            sed -i "s!https://slint.dev/releases/.*/docs/!../../!" $output_path/docs/rust/slint/*.html
 
             for lang in rust cpp node; do
-                mkdir -p snapshots/$target_branch/docs/tutorial/$lang
-                cp -a ../docs/tutorial/$lang/book/html/* snapshots/$target_branch/docs/tutorial/$lang
+                mkdir -p $output_path/docs/tutorial/$lang
+                cp -a ../docs/tutorial/$lang/book/html/* $output_path/docs/tutorial/$lang
             done
-            mkdir -p snapshots/$target_branch/docs/node
-            cp -a ../api/node/docs/* snapshots/$target_branch/docs/node/
-            mkdir -p snapshots/$target_branch/docs/slint
-            cp -a ../target/slintdocs/html/* snapshots/$target_branch/docs/slint/
+            mkdir -p $output_path/docs/node
+            cp -a ../api/node/docs/* $output_path/docs/node/
+            mkdir -p $output_path/docs/slint
+            cp -a ../target/slintdocs/html/* $output_path/docs/slint/
+
+      - name: Adjust redirections
+        if: github.event.inputs.release == 'true'
+        run: |
+            sed -i "/1.0.2/! s,releases/[^/]*/\(.*\),releases/${{ steps.version.outputs.VERSION }}/\1," www/data/_redirects
```

Comment:
This was there in the website job,  this is because there are some lines with 1.0.2 in that file that should NOT be changed.  (yes, i also got confused by that)